docs/i18n.md

```diff
@@ -45,3 +45,13 @@ officially added.
 Login to locize.io to the bitbox-wallet-app project.
 
 Or use a build of the app with an inline editor (see above).
+
+## For developers
+
+New keys can be added to `frontends/web/src/locales/en/app.json`
+
+Keys have to be in alphabetical order, this can be automatically fixed by running
+
+```bash
+npx locize-cli format frontends/web/src/locales --format json
```

Comment:
Can we add this to the scripts/ci.sh? There's already a relevant section in there:
https://github.com/digitalbitbox/bitbox-wallet-app/blob/144eea22c8cf74100fc277ce1ea39d4f89cc0096/scripts/ci.sh#L27-L32

Reply:
Should this simply be `make locize-fix`?

Reply:
and maybe add it to the "Repo maintainers" section above?

Reply:
added "developers" for other contributors that are not maintainers and might run into this issue in the future. do you prefer if it is under "Repo maintainers"?

Reply:
sounds good

Reply:
I mean sounds good as is.

Reply:
replaced with `make locize-fix` and fixed squashed pushed